src/lib/db/client-applications-store.ts

```diff
@@ -334,6 +350,15 @@ export default class ClientApplicationsStore
                 missingFeatures.add(feature_name);
             }
 
+            strategies.forEach((strategy) => {
+                if (
+                    !DEPRECATED_STRATEGIES.includes(strategy) &&
```

Comment:
SDKs need to backwards compatible, so they are sending some very old strategy types that have been removed from newer versions of Unelash. These should not be reported as missing types.